authLoginHelperCreds.user.js

```diff
@@ -346,13 +346,31 @@ function generateCreateUserInput() {
         var validInput = true;
 
         if($(affinityInput).val() == "individual") {
-            var enrolment = $(enrolmentsContainer).children().first().find("input");
+            var _raw_enrolments = $(enrolmentsContainer).children();
+            var _individual_enrolments = [];
 
-            var _key = $($(enrolment)[0]).val();
-            var _name = $($(enrolment)[1]).val();
-            var _value = $($(enrolment)[2]).val();
+            $(_raw_enrolments).each(function() {
+                var _inputs = $(this).find("input");
+
+                var _key = $($(_inputs)[0]).val();
+                var _name = $($(_inputs)[1]).val();
+                var _value = $($(_inputs)[2]).val();
+
+                if(!(nonBlank($(inputField).val()) && nonBlank($(ninoInput).val()) && nonBlank(_key) && nonBlank(_name) && nonBlank(_value))) { validInput = false; }
+
+                _individual_enrolments.push({
+                    key: _key,
+                    name: _name,
+                    value: _value
+                });
+            })
+
+
+            var enrolment = _individual_enrolments[0];
 
-            if(!(nonBlank($(inputField).val()) && nonBlank($(ninoInput).val()) && nonBlank(_key) && nonBlank(_name) && nonBlank(_value))) { validInput = false; }
+            var _key = enrolment.key;
```

Comment:
This could definitely be consolidated, or even removed and the fields called directly in the createNewUser function.